hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileUtil.java

```diff
@@ -2108,4 +2108,21 @@ public static void maybeIgnoreMissingDirectory(FileSystem fs,
     LOG.info("Ignoring missing directory {}", path);
     LOG.debug("Directory missing", e);
   }
+
+  /**
+   * Return true if the FS implements {@link WithErasureCoding} and
+   * supports EC_POLICY option in {@link Options.OpenFileOptions}
```

Comment:
1. add a . at the end of this sentence
2. add a second sentence saying that a message is logged when not available